cli/package.json

```
@@ -120,7 +120,7 @@
     "cypress": "bin/cypress"
   },
   "engines": {
-    "node": "^18.0.0 || ^20.0.0 || >=22.0.0"
+    "node": "^20.0.0 || >=22.0.0"
```

To drop Node.js 23.x support:

```suggestion
    "node": "^20.0.0 || ^22.0.0"
```

To add Node.js 24.x support after this is released on Apr 22, 2025, and then tested with Cypress, that could change to 

```json
    "node": "^20.0.0 || ^22.0.0 || >=24.0.0"
```